charts/argo-cd/templates/argocd-repo-server/deployment.yaml

```diff
@@ -179,7 +179,7 @@ spec:
         name: tmp-dir
       {{- if .Values.repoServer.initContainers }}
       initContainers:
-{{- toYaml .Values.repoServer.initContainers | nindent 6 }}
+      {{- toYaml .Values.repoServer.initContainers | nindent 6 }}
```

Comment:
Changed the indentation here, which is a no-op in the output (thanks to `{{-` plus `nindent`) and seemed to match the convention elsewhere